mode/logicanalyzer.h

```diff
@@ -1,9 +1,10 @@
 void la_test_args (struct command_result *res);
 bool logicanalyzer_setup(void);
+void restart_dma();

```

Comment:
I don't claim to have had time to review this PR fully.  I do have one minor tweak I'd recommend:

It appears this function is only used within `logicanalyzer.c`.  If this is correct, move this declaration to the head of that file, and mark the function as `static`.

Why: Functions only used in the file they are implemented in should be declared `static`, to allow optimizations at compilation (vs. only linking / LTO).

Reply:
Good catch Henry. @DangerousPrototypes , What is the best way to process the fix? Submit another PR? 
Thanks,
Patrick

Reply:
I pushed this fix, thank you both so much.